test/native/test_build_config.cpp

```diff
@@ -4,14 +4,14 @@
  * Unit tests of the build_config.hpp header.
  */
 
-#include "uavcan/uavcan.hpp"
+#include "libuavcan/uavcan.hpp"
 #include "gtest/gtest.h"
 
 /**
  * Test that the UAVCAN_VERSION_MAJOR number is expected.
  */
 TEST(BuildConfigTest, Version) {
-    ASSERT_EQ(2, UAVCAN_LIBRARY_VERSION_MAJOR) << "These tests are designed for version 2 of libuavcan" << std::endl;
+    ASSERT_EQ(2, LIBUAVCAN_VERSION_MAJOR) << "These tests are designed for version 2 of libuavcan" << std::endl;
     ASSERT_EQ(1, UAVCAN_SPECIFICATION_VERSION_MAJOR) << "These tests are designed for version 1 of the UAVCAN protocol" << std::endl;
```

Comment:
I am wondering if we should rename `UAVCAN_SPECIFICATION_VERSION_MAJOR` into `LIBUAVCAN_SPECIFICATION_VERSION_MAJOR`? One might object saying that there is no such thing as "libuavcan specification" though. But the upside of this change is that we're keeping libuavcan-related things properly prefixed.

Reply:
`LIBUAVCAN_UAVCAN_SPECIFICATION_VERSION_MAJOR`

It's long but it's correct.